lib/puppet/provider/mongodb.rb

```diff
@@ -1,3 +1,4 @@
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'util', 'mongodb_output'))
```

Comment:
This feels.. dirty. Maybe implement it like https://github.com/voxpupuli/puppet-mongodb/blob/master/lib/puppet/provider/mongodb_conn_validator/tcp_port.rb#L1